sdk/v1/data.go

```diff
@@ -1,4 +1,4 @@
-package sdk
+package v1
```

Comment:
aren't some of these structs on v2 also?

Reply:
Same type names, different structs. The structs that are equal have been kept in `v1`, the different ones have the same name but are different, so you can instantiate them like: `v1.Integration` or `v2.Integration`.

Reply:
This will be confusing for users.

Reply:
Maybe I'm wrong. The way I understand go package names is that they want to provide meaning to the types they go with. That's why linters complain with constructions like: `metric.MetricSet`.

In this case, I preferred to separate types by package instead of renaming them. IMHO is cleaner having `v1.Integration` and `v2.Integration` (or any other package names) instead of `sdk.Integration` and `sdk.IntegrationProtocol2`.

Reply:
+1